config/environments/development.rb

```diff
@@ -39,9 +39,11 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
-
   config.action_mailer.perform_caching = false
 
+  # Configuration for devise
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
```

Comment:
For developing emails, you may want to check out `mailcatcher`. It's super useful to test emails send as expected, and it's easy to set up. It's used around Shopify a bit, as well.

https://mailcatcher.me/

Reply:
Very interesting. 

I am thinking on creating users based on invitation only, instead of opening for anyone to create simulations.

Reply:
Check out the [`devise_invitable`  gem](https://github.com/scambra/devise_invitable) if you want to use invites + devise.

Reply:
Created issue #28 to add invitation capability.